docs/user/training_a_policy.md

```diff
@@ -0,0 +1,145 @@
+# Training a policy to solve an environment
+This page will guide you training a policy to solve an environment.
+### Define an experiment
+In garage, we train a policy in an experiment, which is a function wrapped by a decorator called `wrap_experiment`. Below is an simple example. `wrap_experiment` could have some arguments. You can see the [experiments doc](experiments.rst) for details of running experiments.
```

Comment:
You should probably mention somewhere here that users should pick a policy that is compatible with the environment's observations and actions (CNN for image observations, discrete policy for discrete action spaces, etc)

Reply:
Good point. Will do.